src/subscription/subscribe.js

```diff
@@ -227,10 +227,10 @@ export function createSourceEventStream(
     const fieldNodes = fields[responseName];
     const fieldNode = fieldNodes[0];
     const fieldDef = getFieldDef(schema, type, fieldNode.name.value);
-    invariant(
-      fieldDef,
-      'This subscription is not defined by the schema.'
-    );
+
+    if (!fieldDef) {
+      return resolve(emptyAsyncIterator());
```

Comment:
This doesn't feel like the right return value for a missing field case.

Reply:
What would be a better one? Presuming an error becomes the domain of a validation step, we should produce something

Reply:
In https://github.com/facebook/graphql/blob/master/spec/Section%206%20--%20Execution.md#executing-requests we describe `Subscribe` as the async variant of `ExecuteQuery`. For `ExecuteQuery`, the spec says, "Return an unordered map containing {data} and {errors}."while `Subscribe` says `Return {responseStream}.`. Perhaps we should update these to be more similar, for example, Subscriptions resolving to an unordered map of {responseStream} and {errors}.